config/rbac/operator-role-knative.yaml

```diff
@@ -35,6 +35,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
This will probably needs to be added in the `helm/camel-k/templates/operator-role.yaml` file.

Reply:
No, it's a leftover. I'll remove in next commit.